README.md

```diff
@@ -32,8 +32,8 @@ Srcbook runs locally on your machine as a CLI application with a web interface.
 
 ### Requirements
 
-- Node.js v20+
-- We recommend using [nvm](https://github.com/nvm-sh/nvm) to manage local node versions
+- [nvm](https://github.com/nvm-sh/nvm) to manage local node versions
```

Comment:
Why did we remove the Node.js 20 requirement?

Reply:
Heavy recommendation to just use nvm, that way the requirement is static and as it changes running nvm will always stay in sync instead of noticing when things randomly break and updating it. I can set it back if you'd prefer

Reply:
I prefer the previous phrasing. You _can_ run srcbook without nvm, and if you want to (maybe you have a good reason to have only one node version), you just want to know node version compatibility.

Reply:
Updated it but keeping the corepack requirement as pnpm 9 isn't true only 9.4 and up support `catalog` so it's reasonable to require corepack and exact pnpm version for guarantee of functionality